Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods'
+gem 'cocoapods', '1.10.0.beta.2'
```

Comment:
Did this have to be hard-coded here to use the beta? `bundle update cocoapods` maybe only works for GA huh?

Reply:
Yes, when using non SerVer variants you need to specify it directly.